.github/workflows/test.yml

```diff
@@ -2,8 +2,16 @@ name: test
 
 on:
   pull_request:
+    paths-ignore:
+      - README.md
+      - CONTRIBUTING.md
+      - docs/**
   push:
     branches: [master]
+    paths-ignore:
+      - README.md
+      - CONTRIBUTING.md
+      - docs/**
```

Comment:
This workflow runs both pre-commit and tests.

Tests are not so important, but pre-commit is for docs.

Also there are tests that test the correct behavior of `docs/scaffolding2copier.sh`, so I think it would be better to use `docs/**.md` instead.

Reply:
Not sure that I understand you about pre-commit. From my understanding pre-commit runs in ``lint.yml``. 

Take a look on new updates, please

Reply:
You're right, I was confused due to some garbage I seem to have copy-pasted from `lint.yml`. I fixed it myself.